src/client/utils/auto-increment-id-generator.js

```diff
@@ -0,0 +1,19 @@
+const MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER || 9007199254740991;
+const MIN_SAFE_INTEGER = Number.MIN_SAFE_INTEGER || -9007199254740991;
+
+export default function createAutoIncrementIdGenerator () {
+    let id = MIN_SAFE_INTEGER;
+
+    return {
+        increment: () => {
+            if (id === MAX_SAFE_INTEGER)
```

Comment:
```js
return id < MAX_SAFE_INTEGER ? ++id : MAX_SAFE_INTEGER;     
```

Reply:
You variant is worse for understanding because one need to remember a unobvious convertation logic. I propose to leave an origin spelling.

Reply:
@churkin What do you think?

Reply:
expected id chain `... 9007199254740990 9007199254740991 -9007199254740991 ...`
but actual with Math.min is `... 9007199254740990 9007199254740991 9007199254740991 ...`

Reply:
```js
id = id === MAX_SAFE_INTEGER ? MIN_SAFE_INTEGER : id + 1;
```

Reply:
ok, got it.